### PR TITLE
r-universe page

### PR DESCRIPTION
**Pull request by maelle**
to add a r-universe page cf #130

### CODE REVIEW

File: content/technotes/2020-11-19-moving-away-travis/index.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ tags:
   - CI
   - Travis
   - packages
+  - r-universe
+summary: Shorter summary hey
```
Comment (maelle) [inline content/technotes/2020-11-19-moving-away-travis/index.md:12]: @jeroen without this the summary = first sentences of the post

**Comment by maelle** [comment]
to see the tech note this PR needs to be rebased

File: themes/ropensci/layouts/partials/skeleton/navbar.html [hunk 2]
```diff
@@ -27,12 +27,6 @@
         <li class="nav-item drp">
           <a class="nav-link" href="/about/">About</a>
           <ul class="sub-menu">
-            <li class="drp">
-              <a>Software Peer Review</a>
```
Comment (maelle) [inline themes/ropensci/layouts/partials/skeleton/navbar.html:31]: software peer review needs to be linked from the about drop-down

File: themes/ropensci/layouts/partials/skeleton/navbar.html [hunk 3]
```diff
@@ -73,12 +76,11 @@
           <a class="nav-link" href="/resources/">Resources</a>
           <ul class="sub-menu">
             <li><a href="/talks-papers/">Talks & Publications</a></li>
+            <li><a class="external-link" href="https://docs.ropensci.org/">Package Docs</a></li>
```
Comment (maelle) [inline themes/ropensci/layouts/partials/skeleton/navbar.html:79]: this makes sense

File: themes/ropensci/layouts/partials/skeleton/navbar.html (hunk 2 above)
Comment (jeroen) [reply themes/ropensci/layouts/partials/skeleton/navbar.html:31]: Oops, thanks.

**Comment by mpadge** [comment]
@jeroen This is great!! Thanks!

**Comment by jeroen** [comment]
Thanks! 😄

**Comment by maelle** [comment]
Btw you both should be featured on your respective project pages somehow?

**Pull request merged**